### PR TITLE
fixing notifications

### PR DESCRIPTION
**Pull request by Redhero1996**
fix paginate, update notification number, display enough number of notification

### CODE REVIEW

File: app/Http/Controllers/User/NotificationController.php [hunk 1]
```diff
@@ -28,23 +28,23 @@ public function getNotifications($limit = -1)
...
-
-        return $data = $this->notification->getNotifications($limit, $with, $where);
+        
+        return $data = $this->notification->getNotifications($with, $where);
     }
 
     public function getAllNotifications()
     {
-        $notifications = $this->getNotifications();
+        $notifications = $this->notification->paginate($this->getNotifications(), config('view.paginate.notifications'));
```
Comment (framgiaciv3) [inline app/Http/Controllers/User/NotificationController.php]: **Reporter**: CHECKSTYLE</br>**Rule**: Generic.Files.LineLength.TooLong</br>**Severity**: warning</br>**File**: app/Http/Controllers/User/NotificationController.php</br>**Line**: 37</br>Line exceeds 120 characters; contains 121 characters

File: app/Http/Controllers/User/NotificationController.php [hunk 2]
```diff
@@ -28,23 +28,23 @@ public function getNotifications($limit = -1)
...
         return view('user.notifications', compact('notifications'));
     }
 
-    public function getLimitNotifications($limit = 0)
+    public function getLimitNotifications()
     {
-        if ($limit < config('view.limit.notifications')) {
-            $notifications = $this->getNotifications(config('view.limit.notifications'));
+        if (count($this->getNotifications()) < config('view.limit.notifications')) {
+            $notifications = $this->getNotifications();
```
Comment (framgiaciv3) [inline app/Http/Controllers/User/NotificationController.php]: **Reporter**: CHECKSTYLE</br>**Rule**: Squiz.WhiteSpace.SuperfluousWhitespace.EndLine</br>**Severity**: error</br>**File**: app/Http/Controllers/User/NotificationController.php</br>**Line**: 45</br>Whitespace found at end of line

File: app/Http/Controllers/User/NotificationController.php [hunk 3]
```diff
@@ -28,23 +28,26 @@ public function getNotifications($limit = -1)
...
-        return $data = $this->notification->getNotifications($limit, $with, $where);
+        
+        return $data = $this->notification->getNotifications($with, $where);
     }
 
     public function getAllNotifications()
     {
-        $notifications = $this->getNotifications();
+        $notifications = $this->notification->paginate(
+            $this->getNotifications(),
```
Comment (framgiaciv3) [inline app/Http/Controllers/User/NotificationController.php]: **Reporter**: CHECKSTYLE</br>**Rule**: Squiz.WhiteSpace.SuperfluousWhitespace.EndLine</br>**Severity**: error</br>**File**: app/Http/Controllers/User/NotificationController.php</br>**Line**: 38</br>Whitespace found at end of line

File: app/Http/Controllers/User/NotificationController.php [hunk 4]
```diff
@@ -28,23 +28,26 @@ public function getNotifications($limit = -1)
         $where = [
             'receive_id' => Auth::id(),
         ];
-
-        return $data = $this->notification->getNotifications($limit, $with, $where);
+
```
Comment (BunnyPi04) [inline app/Http/Controllers/User/NotificationController.php]: dư space

File: app/Http/Controllers/User/NotificationController.php [hunk 5]
```diff
@@ -19,32 +19,34 @@ public function __construct(
...
         $with = [
             'target',
             'userSend',
         ];
         $where = [
             'receive_id' => Auth::id(),
-        ];
-
-        return $data = $this->notification->getNotifications($limit, $with, $where);
+        ];
```
Comment (framgiaciv3) [inline app/Http/Controllers/User/NotificationController.php]: **Reporter**: CHECKSTYLE</br>**Rule**: Squiz.WhiteSpace.SuperfluousWhitespace.EndLine</br>**Severity**: error</br>**File**: app/Http/Controllers/User/NotificationController.php</br>**Line**: 30</br>Whitespace found at end of line

**Pull request merged**